fwl-expect/README.md

```diff
@@ -0,0 +1,92 @@
+# Fewlines Expect
```

Comment:
I would suggest Camp2 Helpers since there will be only one repo for all packages